### PR TITLE
Add buttons to create new facts and parent facts

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

**Comment by beav** [comment]
```diff
+ Coverage   20.08%   20.5%   +0.41%     
```
:+1:

File: src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js [hunk 1]
```diff
@@ -11,15 +11,17 @@ import {
     TableTextInput
 } from '@patternfly/react-inline-edit-extension';
 import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
+import { AddCircleOIcon } from '@patternfly/react-icons';
 
 import { baselinesPageActions } from '../redux';
 import { baselinesTableActions } from '../../BaselinesTable/redux';
-import api from '../../../api.js';
```
Comment (beav) [inline src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js]: what did this do previously?

**Comment by beav** [comment]
i don't see anything egregious here code-wise. testing now

File: src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js (hunk 1 above)
Comment (johnsonm325) [reply src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js]: We were running a patch directly from the api. Now we call an action that calls the api so we can have the data in the payload.

**Comment by beav** [comment]
There are some rough edges but I'd rather address them as bugs than in this PR.

ACK

**Pull request merged**